src/test/scala/com/spotify/ratatool/scalacheck/TableRowGenSpec.scala

```diff
@@ -36,24 +36,40 @@ object TableRowGenSpec extends Properties("TableRowGen") {
     base64a == base64b
   }
 
-  val n = "nullable_fields"
+  val nf = "nullable_fields"
+  val rf = "repeated_fields"
   val richGen = tableRowOf(Schemas.tableSchema)
-    .amend(Gen.choose(10L, 20L))(_.getTableRow(n).set("int_field"))
-    .amend(Gen.choose(10.0, 20.0))(_.getTableRow(n).set("float_field"))
-    .amend(Gen.const(true))(_.getTableRow(n).set("boolean_field"))
-    .amend(Gen.const("hello"))(_.getTableRow(n).set("string_field"))
+    .amend(Gen.choose(10L, 20L))(_.getTableRow(nf).set("int_field"))
+    .amend(Gen.choose(10.0, 20.0))(_.getTableRow(nf).set("float_field"))
+    .amend(Gen.const(true))(_.getTableRow(nf).set("boolean_field"))
+    .amend(Gen.const("hello"))(_.getTableRow(nf).set("string_field"))
```

Comment:
Values in repeated fields should be `java.util.List<T>`, not a single value like this?